src/pages/nonMbLookup/NonMbLookup.tsx

```diff
@@ -8,78 +8,89 @@ import InputWrapper from "./components/InputWrapper";
 import { NAVIGATION_STATE } from "@constants/navigationState";
 import MetaTag from "@components/commons/meta/MetaTag";
 import { useHeader, useModal } from "@hooks";
+import { useLazyPostGuestBookingList } from "@apis/domains/bookings/queries";
 
 const NonMbLookup = () => {
   const navigate = useNavigate();
-
   const { openAlert } = useModal();
   const { setHeader } = useHeader();
+  const { fetchBookingList } = useLazyPostGuestBookingList();
 
-  // 하단 내역 확인 버튼 활성화/비활성화 상태
+  const [formData, setFormData] = useState({
+    bookerName: "",
+    birth: "",
+    number: "",
+    password: "",
+  });
```

Comment:
p2: 
요기 state 의 변수 네이밍을 조금 바꾸면 좋을 거 같아요!

```suggestion
  const [formData, setFormData] = useState({
    name: "",
    birth: "",
    phone: "",
    password: "",
  });
```
이렇게 바꾸면 아래에서 조금 더 간략하게 사용할 수 있어요!